src/components/capture/Create.tsx

```diff
@@ -58,6 +59,10 @@ function CaptureCreate() {
 
     // Supabase stuff
     const supabaseClient = useClient();
+    const { combinedGrants } = useCombinedGrantsExt({
+        onlyAdmin: true,
```

Comment:
I believe `adminOnly` is the more common way to phrase this. A change is not necessary, but calling it out in the event you'd like to update it.